sormas-api/src/main/java/de/symeda/sormas/api/environment/EnvironmentFacade.java

```diff
@@ -11,4 +11,6 @@
 public interface EnvironmentFacade extends CoreFacade<EnvironmentDto, EnvironmentIndexDto, EnvironmentReferenceDto, EnvironmentCriteria> {
 
 	List<EnvironmentDto> getAllEnvironmentsAfter(Date date);
+
+	EnvironmentDto getEnvironmentDataByUuid(String uuid);
```

Comment:
this method exists. please use `getByUuid` of `AbstractBaseEjb`

Reply:
removed